package.json

```diff
@@ -90,6 +95,15 @@
           "default": "",
           "description": "Path to the virtual environment activation script. Use only if you have a custom activation script. It will be sourced using bash before executing Ansible commands. If set, the Interpreter Path setting is ignored."
         },
+        "ansible.vault.executablePath": {
+          "default": "ansible-vault",
+          "description": "%configuration.vault.executablePath%",
```

Comment:
Pls add this to https://github.com/ansible/vscode-ansible/blob/main/package.nls.json

Reply:
Looks like it is already there? https://github.com/ansible/vscode-ansible/blob/5ef7df70ea5834b3b163c37149153027f543ffbf/package.nls.json#L13